rs/rosetta-api/icrc1/ledger/src/main.rs

```diff
@@ -135,18 +142,79 @@ fn pre_upgrade() {
         .expect("failed to write instructions consumed to stable memory");
 }
 
+#[cfg(feature = "upgrade-to-memory-manager")]
```

Comment:
Should we name the feature according to the canister versioning in the design doc, e.g., this would be Canister version 2?

Reply:
This is only a temporary feature used for testing this version's ability to read from memory manager. After this PR is merged, I will delete the feature and this pre_upgrade will become the only pre_upgrade.

Reply:
Sure, but an alternative name could better indicate what this temporary feature stands for, maybe "next-migration-version" or the like would be even better.

Reply:
Wouldn't that be less informative if someone doesn't have access to the design doc?

Reply:
Then maybe: `next-migration-version-memory-manager` or the like?